CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.3.7] - 2023-11-08
+### Fixes
+- Add `@throws` annotations in the PHPDoc blocks throughout the codebase.
```

Comment:
Not sure if this message provides a lot of value to the merchant. Are you adding @throws in order to support a new Php version, or what is the reason?

Reply:
You're right. These changes are meant for developers. They were suggested by [Anders Jenbo](https://github.com/AJenbo) in the below issue.

https://github.com/AltaPay/sdk-php/issues/3